code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -1199,22 +1199,16 @@
 		var/obj/machinery/computer/camera_advanced/xenobio/X = O
 		X.monkeys++
 		to_chat(user, "<span class='notice'>You feed [src] to the [X]. It now has [X.monkeys] monkey cubes stored.</span>")
+		user.drop_item()
```

Comment:
I wonder how many things in the codebase fail to GC because people aren't calling `unEquip` or `drop_item` before `qdel`ing them?